### PR TITLE
Register maine.is-a.dev

### PR DESCRIPTION
**Pull request by Codemaine**
Register maine.is-a.dev with TXT record pointing to vc-domain-verify=maine.is-a.dev,8d16f8af14c09f846ccf.

### CODE REVIEW

**Review by MaskDuck — DISMISSED** [review]
We do not support vercel

File: domains/jermaine.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Codemaine",
+        "email": "jermaineantwi22@gmail.com"
+    },
+    "record": {
+        "TXT": "vc-domain-verify=jay.is-a.dev,ab97cfe7fecbf105ee2c"
```
Comment (21Z) [inline domains/jermaine.json:7]: TXT records alone does nothing to your website... You need a working web page.

File: domains/maine.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "Codemaine",
+           "email": "jermaineantwi22@gmail.com",
+           "discord": "672698075910766622"
+        },
+    
+        "record": {
+            "TXT": "vc-domain-verify=maine.is-a.dev,8d16f8af14c09f846ccf"
```
Comment (21Z) [inline domains/maine.json:9]: TXT records alone does nothing to your website... You need a working web page.

**Review by 21Z — CHANGES_REQUESTED** [review]

**Comment by 21Z** [comment]
> Which hosting providers do you support?

Almost all except vercel and netlify.. You can use Github Pages or Cloudflare Pages

**Comment by Codemaine** [comment]
Alright

**Comment by Codemaine** [comment]
CNAME record pointing to codemaine.github.io

**Comment by 21Z** [comment]
> CNAME record pointing to codemaine.github.io

You can edit the JSON file accordingly

**Comment by Codemaine** [comment]
Which JSON file sorry?

**Comment by 21Z** [comment]
> Which JSON file sorry?

The domain files (jermaine.json and maine.json)

**Comment by Codemaine** [comment]
I've made the necessary changes

File: domains/maine.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "Codemaine",
+           "email": "jermaineantwi22@gmail.com",
+           "discord": "672698075910766622"
+        },
+    
+        "record": {
+            "TXT": "vc-domain-verify=maine.is-a.dev,8d16f8af14c09f846ccf",
```
Comment (21Z) [inline domains/maine.json:9]: Remove the TXT record
```suggestion
```

File: domains/jermaine.json [hunk 4]
```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "Codemaine",
+        "email": "jermaineantwi22@gmail.com"
+    },
+    "record": {
+        "TXT": "vc-domain-verify=jay.is-a.dev,ab97cfe7fecbf105ee2c",
```
Comment (21Z) [inline domains/jermaine.json:7]: Remove the TXT record
```suggestion
```

**Review by 21Z — CHANGES_REQUESTED** [review]

**Review by omsenjalia — DISMISSED** [review]
Cannot access your website

**Comment by Codemaine** [comment]
Fixed: [https://codemaine.github.io](https://codemaine.github.io)

**Review by 21Z — APPROVED** [review]

**Review by iostpa — APPROVED** [review]
Seems good to me now

**Pull request merged**